libcontainer/cgroups/fs/apply_raw.go

```diff
@@ -106,6 +106,13 @@ func (m *Manager) Apply(pid int) (err error) {
 		return nil
 	}
 
+	var c = m.Cgroups
+
+	if c.Paths != nil {
+		m.Paths = c.Paths
+		return cgroups.EnterPid(m.Paths, pid)
+	}
+
```

Comment:
Should we validate these paths before assigning to `m.Paths`? I know they must be valid if they're from Docker, but we might need to support this for runc config file.